core/src/main/java/org/apache/calcite/rel/convert/ConverterRule.java

```diff
@@ -78,7 +78,7 @@ public ConverterRule(Class<? extends RelNode> clazz, RelTrait in,
     super(convertOperand(clazz, predicate, in),
         relBuilderFactory,
         description == null
-            ? "ConverterRule<in=" + in + ",out=" + out + ">"
+            ? "ConverterRule(in:" + in + ",out:" + out + ")"
             : description);
```

Comment:
Why to replace `=` with `:` ?

Reply:
= is not allowed in rule description, it would be better to use :